packages/loot-core/migrations/1704572023730_add_account_sync_source.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE accounts ADD COLUMN account_sync_source TEXT;
+
+UPDATE accounts SET
```

Comment:
@joel-jeremy here is where I updated the migration. Does this look correct to you?

Reply:
This looks good to me but would like to check @MatissJanis's thoughts on this.